src/sentry/tasks/sentry_apps.py

```diff
@@ -246,21 +246,25 @@ def notify_sentry_app(event, futures):
 
 
 def send_webhooks(installation, event, **kwargs):
-    project_ids = Project.objects.filter(
-        organization_id=installation.organization_id,
-    ).values_list('id', flat=True)
-
-    servicehooks = ServiceHook.objects.filter(
-        project_id__in=project_ids,
-    )
+    try:
+        servicehook = ServiceHook.objects.get(
+            organization_id=installation.organization_id,
+            actor_id=installation.id,
+        )
+    except ServiceHook.DoesNotExist:
+        return
 
-    for servicehook in filter(lambda s: event in s.events, servicehooks):
-        if not servicehook.created_by_sentry_app:
-            continue
+    if event not in servicehook.events:
+        return
 
-        if servicehook.sentry_app != installation.sentry_app:
-            continue
+    # The service hook applies to all projects if there are no
+    # ServiceHookProject records. Otherwise we want check if
+    # the event is within the allowed projects.
+    project_limited = ServiceHookProject.objects.filter(
+        service_hook_id=servicehook.id,
+    ).exists()
 
+    if not project_limited:
```

Comment:
Are we just saying that if we are `project_limited` we don't send any webhooks here?

Reply:
currently yeah